src/mme/metrics.c

```diff
@@ -0,0 +1,83 @@
+#include "ogs-app.h"
+#include "mme-context.h"
```

Comment:
I believe you based your patches adding metrics in my patch (28e40a0f1b9048768958344e213c69994be671f5), but you didn't use the latest version of it to use it as a base neither here nor in AMF I believe. Pay attention to some minor updates I did there (I'll mention some below).
I'd welcome if you'd update this one and the AMF one to have all of them in a similar fashion.